src/entities/Note.ts

```diff
@@ -0,0 +1,45 @@
+import { User } from './User';
+import { IsDate } from 'class-validator';
+import { Entity, Column, ManyToOne, JoinColumn } from 'typeorm';
+
+export interface APINote {
+	ownerID: string;
+	targetUserID: string;
+	noteType: NoteType;
+	time: string;
+}
+
+export enum NoteType {
+	Blocked = 0,
+	Liked = 1
+}
+
+@Entity()
+export default class Note {
+	@ManyToOne(() => User, { primary: true })
+	@JoinColumn()
+	public owner!: User;
+
+	@ManyToOne(() => User, { primary: true })
+	@JoinColumn()
+	public targetUser!: User;
+
+	@Column({
+		'type': 'enum',
+		'enum': NoteType
+	})
+	public noteType!: NoteType;
+
+	@Column('timestamptz')
+	@IsDate()
+	public time!: Date;
+
+	public toJSON() {
```

Comment:
```suggestion
	public toJSON(): APINote {
```

Just to make sure that this doesn't accidentally change to not return an APINote in the future.